docker/api/docker-compose.local.yml

```diff
@@ -0,0 +1,19 @@
+# A Compose file with minimal dependencies to be able to run Flagsmith, including its test suite, locally.
+
+name: flagsmith
```

Comment:
This comment, and name seem a little unrelated to the actual content of the docker compose file?

Reply:
How so? The goal is to be able to run e.g. `python manage.py runserver` or `pytest` with environment from `.env-local`, which this Compose file allows.

The name is kept from previous `db.yaml` so the volumes are carried over.

Reply:
But this docker compose file only provides the database component. The comment, and the name (to me) indicate that there should be more services configured here. 

It's not really a huge deal, but it is somewhat confusing to me.

Reply:
> only provides the database component

Which is the only dependency required to run Flagsmith locally, _outside_ of Docker.

Reply:
> _outside_ of Docker

That piece is not clear though. It just states "to run Flagsmith, [...], locally.". 'locally' doesn't imply outside of docker.

Reply:
Agreed. I'll add a comment to clarify.